wercker.yml

```diff
@@ -109,10 +122,90 @@ push:
         username: $OCIRUSERNAME
         password: $OCIRPASSWORD
 
+
+e2e-test:
+  base-path: "/go/src/github.com/oracle/oci-cloud-controller-manager"
+  box:
+    id: iad.ocir.io/oracle/oci-cloud-controller-manager-ci-e2e:1.0.0
+    registry: https://iad.ocir.io/v2
+    username: $OCIRUSERNAME
+    password: $OCIRPASSWORD
+  steps:
+
+    - script:
+      name: set ENV vars
+      code: |
+        export VERSION=$(cat VERSION.txt)
+        echo "${VERSION}"
+
+    - script:
+      name: deploy latest ccm
+      code: make upgrade
+        
+    - script:
+      name: e2e default tests
+      code: make e2e
+    
+    - script:
+      name: rollback original ccm
+      code: make rollback
+
```

Comment:
I cant quite remember how the steps in wercker work. If a step fails, then do the rest of the steps still get run? If so, would the test failing would mean we don't roll back to the old version of the ccm?